src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,75 @@
+package nl.novi.TechItEasy.controllers;
+
+import nl.novi.TechItEasy.exceptions.RecordNotFoundException;
+import nl.novi.TechItEasy.services.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+    private ArrayList<Television> televisions;
+
+    public TelevisionController() {
+        televisions = new ArrayList<>();
+
+        Television t = new Television();
+        t.setName("XboxTV");
+        t.setBrand("Microsoft");
+        t.setPrice(499.99);
+        televisions.add(t);
```

Comment:
al deze logic liever niet in je constructor. 

Als je een fake database wilt maken dan liever:

`public List<Television> TELEVISION_ARRAYLIST = Arrays.asList(
            new Television("AB-1234", "Philips", 499.99),
            new Television("CD-5678", "LG", 399.99)
    );`